apps/common/controllers/address-lookup-loop.js

```diff
@@ -62,7 +42,8 @@ module.exports = class AddressLookupLoopController extends BaseController {
     });
 
     const count = `${formattedlist.length} addresses`;
-    this.options.fields[`${this.field}-address-lookup`].options = [{value: count, label: count}].concat(formattedlist);
+    this.options.fields[`${this.options.locals.field}-address-lookup`].options =
+      [{value: count, label: count}].concat(formattedlist);
```

Comment:
This is going to need some changes to hof-template-mixins to be able to set dynamic options on a select/radio group. I will be picking that work up.